kafka/adminapi.go

```diff
@@ -622,6 +629,10 @@ func (a *AdminClient) waitResult(ctx context.Context, cQueue *C.rd_kafka_queue_t
 
 // cToTopicResults converts a C topic_result_t array to Go TopicResult list.
 func (a *AdminClient) cToTopicResults(cTopicRes **C.rd_kafka_topic_result_t, cCnt C.size_t) (result []TopicResult, err error) {
+	err = a.verifyClient()
```

Comment:
Not required to check this in non-exposed/non-public functions (here and elsewhere)

Reply:
All the conversion c->go, go->c methods etc all don't need this